simple_parsing/wrappers/field_wrapper.py

```diff
@@ -1042,3 +997,47 @@ def get_argparse_options_for_annotation(t: Type) -> Dict[str, Any]:
     #     else:
     #         _arg_options["nargs"] = "*"
     return _arg_options
+
+
+def _optional_args(_arg_options, type_arguments):
```

Comment:
I agree that the code block for optional args was a bit ugly, but I don't think it's super relevant to this PR to move it here. Plus, I'm not really able to tell if anything was changed in that block. Would you mind reverting this?

Reply:
I'm using it in two places now, would you prefer it if I duplicated the code?

It is hard to see the changes but basically there are none (except for passing in type_arguments as a parameter)

Reply:
Oh alright then. Didn't spot that.